tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -511,7 +512,10 @@ public function testEnsureTableOptionsAreReflectedInMetadata() : void
         self::assertEquals('utf8_general_ci', $onlineTable->getOption('collation'));
         self::assertEquals(42, $onlineTable->getOption('autoincrement'));
         self::assertEquals('This is a test', $onlineTable->getOption('comment'));
-        self::assertEquals(['row_format' => 'COMPRESSED'], $onlineTable->getOption('create_options'));
+        self::assertEquals([
+            'row_format' => 'COMPRESSED',
+            'partitioned' => true,
```

Comment:
Would it make sense to split this into a further test?

Reply:
So that two different tests, if failed, would show which kind of options we're failing to parse? I don't know. Parsing a mix of different options at a time is also a valid case. I'd leave it as is.